front/src/modules/simulationResult/components/SpaceTimeChart/useGetProjectedTrainOperationalPoints.ts

```diff
@@ -9,16 +11,24 @@ import {
   type PathProperties,
   type TrainScheduleResult,
 } from 'common/api/osrdEditoastApi';
+import { useOsrdConfSelectors } from 'common/osrdContext';
+
+const isStation = (chCode: string): boolean => chCode === 'BV' || chCode === '00' || chCode === '';
```

Comment:
Thanks for this utils function :heart_hands: 
Maybe you can move it in modules/pathfinding ?

Reply:
Done